### PR TITLE
[Copy] Framework support is out of beta

### PR DESCRIPTION
**Pull request by mrackwitz**
:tada:

### CODE REVIEW

**Comment by segiddins** [comment]
:+1:

**Comment by orta** [comment]
Feels good to me too :+1:

File: lib/cocoapods/installer.rb [hunk 1]
```diff
@@ -421,9 +421,8 @@ def verify_framework_usage
 
           swift_pods = pod_targets.select(&:uses_swift?)
           unless swift_pods.empty?
-            raise Informative, 'Pods written in Swift can only be integrated as frameworks; this ' \
-              'feature is still in beta. Add `use_frameworks!` to your Podfile or target to opt ' \
-              'into using it. ' \
+            raise Informative, 'Pods written in Swift can only be integrated as frameworks; ' \
+              'Add `use_frameworks!` to your Podfile or target to opt into using it. ' \
```
Comment (segiddins) [inline lib/cocoapods/installer.rb]: shouldn't be capitalized after a semicolon.

**Comment by neonichu** [comment]
:+1:

**Pull request merged**